x/ibc/02-client/types/state.go

```diff
@@ -4,7 +4,7 @@ package types
 // Any actor holding the Stage can access on and modify that client information.
 type State struct {
 	// Client ID
-	id string
+	Id string
```

Comment:
It seems better to refactor the func `GetClientState` to set the `id`, but this change is smaller, I'm not sure about your preference.